src/App.vue

```diff
@@ -94,7 +94,6 @@ export default defineComponent({
           'eComStore': this.userProfile.stores.find((store: any) => store.productStoreId === event['detail'].value)
         })
       }
-      emitter.emit("productStoreChanged")
```

Comment:
Revert unwanted changes.

Reply:
Actually this "emitter.emit("productStoreChanged")" was giving error that's why I removed it & I checked on settings page & job-manager app, we are not emmiting "productStoreChanged" on setEComStore method. Will you please let me know why we need to emit productStoreChanged on logout?

Current status - resolved merge conflit.